.babelrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": 100
+        }
```

Comment:
Shouldn't this be the same as https://github.com/zendeskgarden/react-components/blob/4d7f950f6cfaf99a87fbb4bd66eb1ee3818191bf/babel.config.js#L13?

Reply:
Correct! Will update.